### PR TITLE
Fix SVG processing

### PR DESCRIPTION
**Pull request by 343dev**

### CODE REVIEW

File: lib/convert.js [hunk 1]
```diff
@@ -66,17 +66,20 @@ async function convert({ paths, lossless, avif, webp, force }) {
...
-  tasksResult.forEach(({ fileBuffer, filePath, outputFormat } = {}) => {
-    const fileSize = {
-      before: fs.statSync(filePath).size,
-      after: fileBuffer.length,
-    };
-
-    totalSize.before += fileSize.before;
-    totalSize.after += fileSize.after;
+  tasksResult
+    .filter(Boolean)
```
Comment (343dev) [inline lib/convert.js:70]: undefined/null values are now filtered out of the tasksResult array.

Now script execution will not stop at this point if an error occurs 🤞

File: svgo/config.js [hunk 2]
```diff
@@ -50,5 +50,7 @@ module.exports = {
     'convertEllipseToCircle',
     'sortAttrs',
     'sortDefsChildren',
+    'removeOffCanvasPaths',
+    'reusePaths',
```
Comment (343dev) [inline svgo/config.js:54]: Just added some useful plugins.

File: svgo/removeUnknownsAndDefaults.js [hunk 3]
```diff
@@ -1,132 +1,225 @@
 /* eslint-disable */
```
Comment (343dev) [inline svgo/removeUnknownsAndDefaults.js:1]: Last time I forgot to update this file when updating SVGO to the latest version and it caused the application to crash when processing SVGs.

File: lib/convert.js (hunk 1 above)
Comment (igoradamenko) [reply lib/convert.js:70]: 🤔

**Review by igoradamenko — APPROVED** [review]

**Comment by igoradamenko** [comment]
Published @funboxteam/optimizt@2.7.4.

**Pull request merged**